interpretation/src/main/kotlin/com/hartwig/actin/treatment/TreatmentRankingModel.kt

```diff
@@ -21,12 +24,27 @@ data class TreatmentRankResult(val treatment: String, val scores: List<EvidenceS
     }
 }
 
-data class TreatmentEvidenceWithTarget(val treatmentEvidence: TreatmentEvidence, val target: String)
+data class TreatmentEvidenceWithTarget(val treatmentEvidence: TreatmentEvidence, val target: String, val isIndirect: Boolean)
 data class DuplicateEvidenceGrouping(val treatment: String, val gene: String?, val tumorMatch: TumorMatch, val benefit: Boolean)
 
-class TreatmentRankingModel(private val scoringModel: EvidenceScoringModel) {
+class TreatmentRankingModel(
+    private val scoringModel: EvidenceScoringModel,
+    private val includeIndirectTreatmentEvidence: Boolean = true
```

Comment:
we can probably get rid of the feature toggle sooner than later, it's an experimental feature anyway